runtime/src/main/kotlin/kotlin/native/Platform.kt

```diff
@@ -81,6 +81,14 @@ public object Platform {
     public val isDebugBinary: Boolean
         get() = Platform_isDebugBinary()
 
+    /**
+     * If the memory leak checker is activated, by default true in debug mode, false in release.
+     * When memory leak checker is activated, and leak detected on process termination process
+     * is terminated with non-zero exit code.
```

Comment:
Is it?

Reply:
Oh, forgot to push update doc.

Reply:
Error message without termination is somewhat indiscoverable (including leaks in Kotlin/Native tests). Why not terminate?

Reply:
Well, OK, will update message as well.